### DISCUSSION TITLE
Battery Alert, SSID field and payload

### DISCUSSION THREAD

**Discussion by micha53** [meatpiHQ/wican-fw#291] [Q&A]
Who knows what the SSID within the Battery Alert is used for and what has to be filled in here? My home network or something other. Must it get filled only when not the home network is used or in each case?
And what is the content of the Payload when an alert occures?

**Comment by meatpiHQ** [comment]
@micha53 in case you did not find the answer yet. Yes it's your home network or whatever network you want it to connect to when sending the alerts.

**Reply by micha53** [reply]
Is the station config data default when I leave the field blank?

**Reply by meatpiHQ** [reply]
No it will fail to connect
